src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -62,6 +81,10 @@ public class Utils {
   public static final String SF_SSL = "sfssl"; // for test only
   public static final String SF_WAREHOUSE = "sfwarehouse"; // for test only
   public static final String PRIVATE_KEY_PASSPHRASE = "snowflake.private.key" + ".passphrase";
+  public static final String SF_AUTHENTICATOR = "snowflake.authenticator";
+  public static final String SF_OAUTH_CLIENT_ID = "snowflake.oauth.client.id";
+  public static final String SF_OAUTH_CLIENT_SECRET = "snowflake.oauth.client.secret";
+  public static final String SF_OAUTH_REFRESH_TOKEN = "snowflake.refresh.token";
```

Comment:
thinking out loud: do we need oauth here as well to be consistent? since it only applies to oauth
`snowflake.oauth.refresh.token`

Reply:
Forgot to add oauth here. Added oauth prefix since` cliend_id` might seems misleading, what do you think?

Reply:
makes sense to me